common/src/objects/state.rs

```diff
@@ -74,6 +85,13 @@ impl TryFrom<ank_base::State> for State {
                 .into_iter()
                 .map(|(k, v)| Ok((k.to_owned(), v.try_into()?)))
                 .collect::<Result<HashMap<String, StoredWorkloadSpec>, String>>()?,
+            configs: item
```

Comment:
Its Friday evening, but I am currently thinking about these lines of code: Why is the `item.workloads` call above (I cannot select it in the web page since it was not changed) throwing an error if it is missing? Couldn't be the workloads optional as well in the proto message?

Reply:
You are right. I have also changed it for workloads.